stories/molecules/components/stepper/Stepper.stories.mdx

```diff
@@ -711,9 +1352,46 @@ By default the label is provided by the defaults translation files (see the Inte
           )
         }
       ]
+      const { state, dispatch } = useStepper(
+        'q4',
+        steps.map(step => ({ id: step.id, status: step.status }))
+      )
+      const handlePrevious = useCallback(() => {
+        dispatch({
+          type: 'previousClicked'
+        })
+      }, [dispatch])
+      const handleNext = useCallback(() => {
+        dispatch({
+          type: 'stepCompleted'
+        })
+      }, [dispatch])
+      const handleSubmit = useCallback(() => {
+        dispatch({
+          type: 'stepperSubmitted'
+        })
+      }, [dispatch])
+      const handleReset = useCallback(() => {
```

Comment:
On reset the q2 and q3 statuses stay completed until they're the current step. Please add reset to all statuses when pressing the reset button for expected behaviour 😃